src/js/components/CheckboxTable.js

```diff
@@ -79,22 +79,27 @@ class CheckboxTable extends React.Component {
   renderHeadingCheckbox() {
     let checked = false;
     let indeterminate = false;
-    let {allowMultipleSelect, checkedItemsMap, disabledItemsMap, data} = this.props;
+    let {
+      allowMultipleSelect,
+      checkedItemsMap,
+      disabledItemsMap,
+      data
+    } = this.props;
 
     if (!allowMultipleSelect) {
       return null;
     }
 
-    let checkedItemsCount = Object.keys(checkedItemsMap).length;
-    let disabledItemsCount = Object.keys(disabledItemsMap).length;
+    let checkedCount = Object.keys(checkedItemsMap).length;
```

Comment:
I'm ok with this change but don't think that shortening variables names is generally a good fix to obey to the 80 characters rule.